pandas/tests/sparse/test_format.py

```diff
@@ -1,135 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import print_function
```

Comment:
was this on purpose?

Reply:
No idea what happened here.  Will restore.